.github/workflows/link-check-all.yml

```diff
@@ -0,0 +1,23 @@
+name: Link Check All
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  run:
+    name: Link Check All
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Run Link Check
+      uses: "iterative/link-check.action@master"
+      with:
+        source: "filesystem"
+        configFile: "config/link-check/config.yml"
+        reportUnusedPatterns: true
+        output: >-
+          ["consoleLog", "exitCode"]
```

Comment:
we mix two different abstractions here to my mind- actual output + exit code. It might be quite confusing for users.

Reply:
I would recommend to keep exit code setting a separate explicit option

Reply:
something similar to linters - failOnError: true/false? we might even include it into both to make it extra explicit

Reply:
Sure, we can just treat the exit code as a special output controlled by a flag. It's already hard-coded to always run last, so it needs special treatment one way or another.

Reply:
The exit code is now separate from the output strategies, always applying on CLI and the CLI mode of the GHA